### PR TITLE
updating the documentation to show correct environments

### PR DESCRIPTION
**Pull request by ivlists**
updating the documentation to showcase correct environments where this functionality is available.

### CODE REVIEW

File: Teams/alerts/teams-admin-alerts.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ Your organization can use Teams monitoring and alerting to do the following item
 - Take corrective actions to get things back on-track.
 
 > [!NOTE]
-> The alert functionality within the Teams admin center is not available in GCC/GCC-High environments.
+> The alert functionality within the Teams admin center is available in Public and GCC environments only. Its not available in other environments yet.
```
Comment (JohanFreelancer9) [inline Teams/alerts/teams-admin-alerts.md:33]:
```suggestion
> The alert functionality within the Teams admin center is only available in Public and GCC environments. It is not yet available in other environments.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @ivlists, kindly commit the suggested change. Many thanks.

**Review by ivlists — COMMENTED** [review]
reviewed

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@MikePlumleyMSFT 
PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Pull request merged**